### ISSUE TITLE
How can I get the final model found in Neural Architecture Search?

### ISSUE THREAD

**Issue by CppMaster** [microsoft/nni#3171]
**Environment**:
- NNI version: 1.9
- NNI mode (local|remote|pai): local
- Client OS: Red Hat 7.9
- Server OS (for remote mode only):
- Python version: 3.7.6
- PyTorch/TensorFlow version: torch==1.40, tensorflow==2.1.0
- Is conda/virtualenv/venv used?: yes
- Is running in Docker?: no

**What issue meet, what's expected?**:
Hi!
I've run an example NAS script: https://github.com/microsoft/nni/blob/v1.9/examples/nas/enas/search.py
It found a model with accuracy about 65%, so as expected. How can I get the model as a Keras model (or PyTorch if Keras is impossible), so I can use it? Or at least just architecture with random weights. I've tried to search for an answer in the documentation or in the code, but I failed to do so. I probably need to get a Graph somehow to call 'produce_keras_model' function, but how can I do it?

**Comment by ultmaster** [comment]
You will get an architecture json after the script has been run. You need to run "retrain" the model with the json.

However, I checked, The retrain code of ENAS hasn't been finished yet. It should be pure PyTorch logic and not hard to implement. See #3079.

**Comment by zuimeiyujianni** [comment]
I have implemented a simple retrain code of ENAS. I use Pytorch.(only micro search)
